React/Base/RCTConvert.m

```diff
@@ -614,11 +616,114 @@ + (type)type:(id)json                                       \
       @"systemPurpleColor": @{}, // 10_10
       @"systemRedColor": @{},    // 10_10
       @"systemYellowColor": @{}, // 10_10
+#else // TARGET_OS_IOS
+      // https://developer.apple.com/documentation/uikit/uicolor/ui_element_colors
+      // Label Colors
+      @"labelColor": @{ // 13
+        RCTFallbackARGB: @(0xFF000000)
+      },
+      @"secondaryLabelColor": @{ // 13
+        RCTFallbackARGB: @(0x993c3c43)
+      },
+      @"tertiaryLabelColor": @{ // 13
+        RCTFallbackARGB: @(0x4c3c3c43)
+      },
+      @"quaternaryLabelColor": @{ // 13
+        RCTFallbackARGB: @(0x2d3c3c43)
+      },
+      // Fill Colors
+      @"systemFillColor": @{ // 13
+        RCTFallbackARGB: @(0x33787880)
+      },
+      @"secondarySystemFillColor": @{ // 13
+        RCTFallbackARGB: @(0x28787880)
+      },
+      @"tertiarySystemFillColor": @{ // 13
+        RCTFallbackARGB: @(0x1e767680)
+      },
+      @"quaternarySystemFillColor": @{ // 13
+        RCTFallbackARGB: @(0x14747480)
+      },
+      // Text Colors
+      @"placeholderTextColor": @{ // 13
+        RCTFallbackARGB: @(0x4c3c3c43)
+      },
+      // Standard Content Background Colors
+      @"systemBackgroundColor": @{ // 13
+        RCTFallbackARGB: @(0xFFffffff)
+      },
+      @"secondarySystemBackgroundColor": @{ // 13
+        RCTFallbackARGB: @(0xFFf2f2f7)
+      },
+      @"tertiarySystemBackgroundColor": @{ // 13
+        RCTFallbackARGB: @(0xFFffffff)
+      },
+      // Grouped Content Background Colors
+      @"systemGroupedBackgroundColor": @{ // 13
+        RCTFallbackARGB: @(0xFFf2f2f7)
+      },
+      @"secondarySystemGroupedBackgroundColor": @{ // 13
+        RCTFallbackARGB: @(0xFFffffff)
+      },
+      @"tertiarySystemGroupedBackgroundColor": @{ // 13
+        RCTFallbackARGB: @(0xFFf2f2f7)
+      },
+      // Separator Colors
+      @"separatorColor": @{ // 13
+        RCTFallbackARGB: @(0x493c3c43)
```

Comment:
Maybe I missed the comment somewhere, but all these hex values came from where?

Reply:
They were generated some logging breakpoints I added to a unit test.  They are the RGBA values resolved from the semantic colors on iOS 13 in light mode.

Reply:
Adding comment